DeltaCore/UI/Controller/ControllerView.swift

```diff
@@ -292,11 +293,18 @@ public extension ControllerView
         self.buttonsView.controllerSkin = self.controllerSkin
         self.buttonsView.controllerSkinTraits = self.controllerSkinTraits
         
+        var debugModeEnabled = false
         if let isDebugModeEnabled = self.controllerSkin?.isDebugModeEnabled
         {
-            self.controllerDebugView.isHidden = !isDebugModeEnabled
+            debugModeEnabled = isDebugModeEnabled
+        }
+        if self.isControllerSkinDebugModeEnabled == true
```

Comment:
nit: I prefer not comparing Bools to true for simple if statements

```suggestion
        if self.isControllerSkinDebugModeEnabled
```